tsconfig.test.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Separate tsconfig for test folder, as build command should only output results for lib folder to dist.

Reply:
Not sure if it works fine this way with the inline checker in PHPStorm. Will check once in master.